### DISCUSSION TITLE
You are not allowed to reconfigure SEB

### DISCUSSION THREAD

**Discussion by dbuechel** [SafeExamBrowser/seb-win-refactoring#753] [Q&A]
Hello, I'm back again and i thought my SEB issue was fixed, but my previous issue was fixed and now i'm having an issue with "You are not allowed to reconfigure SEB." when i tried to enter my test.

Btw, on Reset utility it says 
Failed to reset 1 items!
 - EaseOfAccessConfiguration
 (tried to restore the default values it says same thing)
 
I don't know how to fix the "You are not allowed to reconfigure SEB" i tried to reinstall SEB and it still giving me same not allowed, hope you can help me, thank you.

_Originally posted by @CyrusSE in https://github.com/SafeExamBrowser/seb-win-refactoring/discussions/729#discussioncomment-7821134_

**Comment by dbuechel** [comment]
That means you are using a SEB configuration which does not allow reconfiguration (see also https://safeexambrowser.org/windows/win_usermanual_en.html#ExamPane > Reconfiguring Secure / Exam Session). There is no need to reinstall SEB nor use the Reset Utility, simply either delete the local client configuration (if it is not required) or then contact the exam provider, as they'll need to enable this option if students should be able to reconfigure SEB.

**Reply by CyrusSE** [reply]
How do i delete "local client configuration" and i don't think i can contact the exam provider to do that because my friends doesn't have this problem when they try to do the exam.

**Reply by dbuechel** [reply]
Delete the configuration file `%AppData%\SafeExamBrowser\SebClientSettings.seb`.

**Reply by CyrusSE** [reply]
in my `%AppData%\SafeExamBrowser\` there is only `SebConfig.log` and there is no `SebClientSettings.seb` in it.

**Reply by dbuechel** [reply]
Then you are using a SEB configuration which does not allow reconfiguration, and the setting needs to be changed respectively in that configuration.

**Reply by dbuechel** [reply]
Please do provide the log files of the affected session(s), they can be found under `%LocalAppData%\SafeExamBrowser\Logs`.

**Reply by CyrusSE** [reply]
[2023-12-12_21h14m29s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/13649149/2023-12-12_21h14m29s_Runtime.log)
[2023-12-12_21h14m29s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/13649201/2023-12-12_21h14m29s_Client.log)

Here are the log files.

**Reply by dbuechel** [reply]
As already elaborated, the configuration you're starting by double-clicking is `C:/Users/ICAL/Downloads/config (24).seb`, and it does not allow reconfiguration:

```
2023-12-12 21:14:57.137 [28] - INFO: Denied download request for configuration file 'config.seb'.
```
